tensor2tensor/models/transformer.py

```diff
@@ -863,9 +863,15 @@ def symbols_to_logits_fn(ids, i, cache):
         vocab_size = tf.shape(ret)[1]
 
         def forced_logits():
+          # Workaround for: tf.one_hot(
+          #               tf.repeat(partial_targets[:, i], [beam_size]), vocab_size, 0.0,
+          #               -1e9)
+          # Can be replaced by the above in future versions (from tf 1.15).
```

Comment:
Hi @senarvi -- thanks for the PR, I'm going to be releasing on TF 1.15 really soon.

Can you "flip" the comments, i.e. make the 1.15 one main and put the other in a comment?

Also a brief explanation of why this workaround is needed would be great